tests/wpt/metadata/resource-timing/idlharness.any.js.ini

```diff
@@ -44,12 +44,18 @@
   [PerformanceResourceTiming interface: resource must inherit property "domainLookupStart" with the proper type]
     expected: FAIL
 
+  [PerformanceResourceTiming interface: resource must inherit property "connectStart" with the proper type]
+    expected: FAIL
```

Comment:
Please remove this particular test result, since it passes with your changes.

Reply:
Ok, sorry for the mistake.
So if I understood it well, this is the changes I need to make:
```
diff --git a/tests/wpt/metadata/resource-timing/idlharness.any.js.ini b/tests/wpt/metadata/resource-timing/idlharness.any.js.ini
index 46f3c03ddc..1938b2ca3f 100644
--- a/tests/wpt/metadata/resource-timing/idlharness.any.js.ini
+++ b/tests/wpt/metadata/resource-timing/idlharness.any.js.ini
@@ -44,18 +44,12 @@
   [PerformanceResourceTiming interface: resource must inherit property "domainLookupStart" with the proper type]
     expected: FAIL
 
-  [PerformanceResourceTiming interface: resource must inherit property "connectStart" with the proper type]
-    expected: FAIL
-
   [PerformanceResourceTiming interface: attribute workerStart]
     expected: FAIL
 
   [PerformanceResourceTiming interface: attribute domainLookupStart]
     expected: FAIL
 
-  [PerformanceResourceTiming interface: attribute connectStart]
-    expected: FAIL
-
   [PerformanceResourceTiming interface: resource must inherit property "redirectEnd" with the proper type]
     expected: FAIL
```

Reply:
Correct.

Reply:
Should be good now.